### PR TITLE
debian cadvisor package created

### PR DESCRIPTION
**Pull request by rajesh-rajagopal**

### CODE REVIEW

File: reprepro-new.sh [hunk 1]
```diff
@@ -94,6 +94,34 @@ mv /var/repo/1.5/ubuntu/14.04/testing /var/www/html/repo/1.5/ubuntu/14.04
...
+find ./jessie -name \*.deb -exec reprepro --ask-passphrase -Vb /var/repo/1.5/debian/8.5/testing includedeb jessie {} \;
+mv $(find /var/www/html/repo/1.5/debian/8.5/testing/pool/testing -name *.deb) /var/www/html/arkave
+rm -r /var/www/html/repo/1.5/debian/8.5/testing
+
+mv /var/repo/1.5/debian/8.5/testing /var/www/html/repo/1.5/debian/8.5
+chmod -R ugo+rX /var/www/html/repo/1.5/debian/8.5
+
+}
+
+
```
Comment (indykish) [inline reprepro-new.sh]: @rajesh-rajagopal  can you delete aptly.sh ?

File: megamcadvisor/jessie/g [hunk 2]
```diff
@@ -2,19 +2,10 @@
...
-cp -rf ./conf/ ../../../../../usr/share/megam/cadvisor
-cp cadvisor ../../../../../usr/share/megam/cadvisor/bin
-
-cd ../../../../../
-
-cp riakstash ./usr/share/megam/cadvisor/bin
-cp riakstash.cron.d ./usr/share/megam/cadvisor/bin
-
-fpm -s dir -t deb -n $1 -v $2 -d "verticecommon, unzip" --before-install ./preinst --after-install ./postinst --before-remove ./prerm --after-remove ./postrm  --license "Apache" --vendor "Megam Systems" --maintainer "<rajthilak@megam.io>" --url "https://www.megam.io/" --description "The engine for Megam" ./usr/share/megam/cadvisor
+fpm -s dir -t deb -n $1 -v $2 -d "verticecommon, unzip" --before-install ./preinst --after-install ./postinst --before-remove ./prerm --after-remove ./postrm  --deb-init ./etc/systemd/system/cadvisor.service --license "Apache" --vendor "Megam Systems" --maintainer "<rajthilak@megam.io>" --url "https://www.megam.io/" --description "The engine for Megam" ./usr/share/megam/cadvisor
```
Comment (indykish) [inline megamcadvisor/jessie/g]: @rajesh-rajagopal 
- Change the description to "Analyzes resource usage and performance characteristics of running containers/machine."
- maintainer "Megam Maintainer <info@megam.io>"  `for all `

File: verticenilavu/nix/g [hunk 3]
```diff
@@ -8,7 +8,7 @@ bundle install
...
+fpm -s dir -t deb -p "$MEGAM_HOME"  -f  --iteration $3  -n $1 -v $2 -d "verticecommon,git-core,curl,bundler,rake,zlib1g-dev,build-essential,libssl-dev,libreadline-dev,libyaml-dev, libsqlite3-dev,sqlite3,libxml2-dev,libxslt1-dev,libcurl4-openssl-dev,python-software-properties,ruby2.3.1,ruby2.3.1-dev,wbritish,wamerican" --before-install ./preinst --after-install ./postinst --before-upgrade ./preupgrd --before-remove ./prerm --deb-upstart ./etc/init/verticenilavu --license "Apache V2" --vendor "Megam Systems" --category "infrastructure" --maintainer "Thomas Alrin<thomasalrin@megam.io>, Raj Thilak <rajthilak@megam.io>, Kishorekumar Neelamegam<nkishore@megam.io>, Ranjitha<ranjithar@megam.io>, Vijaykanth<mvijaykanth@megam.io>" --url "http://docs.megam.io/docs/vertice" --description "The userinterface using Ruby on Rails which provides the one click launcher for
```
Comment (indykish) [inline verticenilavu/nix/g]: - Change maintainer

File: reprepro-new.sh (hunk 1 above)
Comment (rajesh-rajagopal) [inline reprepro-new.sh]: @indykish now i can change the maintainer as "Megam Maintainer info@megam.io" for all the files
and delete the aptly.sh file.

File: verticenilavu/nix/g (hunk 3 above)
Comment (indykish) [inline verticenilavu/nix/g]: @rajesh-rajagopal  Can you check the maintainer ? `verticenilavu` still has a big list.

**Comment by indykish** [comment]
@rajesh-rajagopal  Its look good, merging it. `jessie` will help us unravel/will be the ticket to  `16.04`

**Pull request merged**